### ISSUE TITLE
Unable to run on server

### ISSUE THREAD

**Issue by ozh** [YOURLS/YOURLS#284]
This is a **COPY** of **Issue 284: [Unable to run on server](http://code.google.com/p/yourls/issues/detail?id=284)**, filed on Google Code before the project was [moved on Github](https://github.com/ozh/google-issues-to-github).
- Submitted on 2010-03-23T17:41:28.000Z by [5cent...@googlemail.com](http://code.google.com/u/110294585501559549549/)
- Status: Invalid

Please **review** the original issue and especially its comments. **Comments here on closed issues will be ignored**. Thanks.
## Original description

```
Hi

I have installed and the script shows only when i go to www.domain.com/admin
I want to allow users to utilize the service but when I visit
www.domain.com it shows nothing as there is no index.php file in the
downloaded package.

can you shed some light on this please?

Basically i want to start similar to bit.ly
a user shorten the url then he will be prompted with the short url as well
as a link to check the stats...

thanks
```
